.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: rust
 sudo: false
 rust:
```

Comment:
Ah ok, would it be possible to leverage `mem::transmute` for now then? It's fine to just build the library on 1.0.0 instead of running the full test suite which may require more features.